graphics/libjpeg/Portfile

```diff
@@ -0,0 +1,34 @@
+# -*- coding: utf-8; mode: tcl; tab-width: 4; indent-tabs-mode: nil; c-basic-offset: 4 -*- vim:fenc=utf-8:ft=tcl:et:sw=4:ts=4:sts=4
+
+PortSystem          1.0
+PortGroup           github 1.0
+
+github.setup        thorfdbg libjpeg c719010a26ce0c666e98b2acf924ad5fc24b4f5d
+version             2024.11.05
+revision            0
+categories          graphics
+license             GPLv3
+maintainers         nomaintainer
+
+description         Complete implementation of 10918-1 (JPEG) from jpeg.org
+
+long_description    A complete implementation of 10918-1 (JPEG) coming \
+                    from jpeg.org (the ISO group) with extensions for \
+                    HDR standardized as 18477 (JPEG XT).
+
+checksums           rmd160  245dc39bc712770499a86d904e8aa332a2118c1a \
+                    sha256  62e8c8b1898a7e9195c90b673e6db0090b474372b4e7f485e471f37a08852e18 \
+                    size    560772
+
+use_autoreconf      yes
+
+depends_build-append \
+                    port:autoconf \
+                    port:automake
```

Comment:
see `port lint` warning; shouldn't these dependency be added already by setting `use_autoreconf yes` - please check.

Reply:
Thanks, done.